services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -430,6 +430,12 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
 
     DIRECTOR_V2_DOCKER_REGISTRY: RegistrySettings = Field(auto_default_from_env=True)
 
+    DIRECTOR_V2_SERVICES_CUSTOM_CONSTRAINTS: str = Field(
+        "",
+        description="added to service Placement constraint in dynamic-sidecars",
+        example="node.labels.region==east",
```

Comment:
Q: did you try putting more than one constraint?

Reply:
I've added a regex and made it singular, only one constraint is accepted.

Reply:
well actually it might cool to put several ones...

Reply:
Here is an example on how it could be done combining regex and and list of strings

https://github.com/pcrespov/sandbox-python/blob/8e6312bc1fbbddeecbac4df419d79f09c4562106/pydantic-lib/test_settings.py#L7

Reply:
Thanks for the nice suggestion!